index.html

```diff
@@ -10,219 +10,268 @@
             </div>
+
             </div>
-            </section>
-        </section>
-        <section class="main">
-            <div class="about-myself">
-                <h2 class="about">About Myself</h2>
-                <p class="text">
-                Hello I’m a software developer! I can help you build a product , feature
-                or website Look through some of my work and experience! If you like what
-                you see and have a project you need coded, don’t hestiate to contact me.
-                </p>
-                <h2 class="hex connect">LET’S CONNECT</h2>
-                <ul class="social-media">
-                <li clalss="socials">
-                <a href="#"><img src="images/socials/twitter.svg" /></a>
-                </li>
-                <li clalss="socials">
-                <a href="#"><img src="images/socials/Linkedin.svg" /></a>
-                </li>
-                <li clalss="socials">
-                <a href="#"><img src="images/socials/m.svg" /></a>
-                </li>
-                <li clalss="socials">
-                <a href="#"><img src="images/socials/Vector1.svg" /></a>
-                </li>
-                <li clalss="socials">
-                <a href="#"><img src="images/socials/Vector2.svg" /></a>
-                </li>
-                </ul>
-                <div class="see-project">
-                <button type="button" class="hex">Get my resume</button>
-                </div>
-                <div class="languages-list">
+            <div class="languages-list">
                 <ul>
-                <li class="lang-title">
-                <p class="langusge">Languages</p>
-                <img class="allow-down" src="images/Icons/vector.png" alt="allow-down" />
-                </li>
-                <li class="lang">
-                <img src="images/langusges/js.png" alt="js" />
-                <p>JavaSript</p>
-                </li>
-                <li class="lang">
-                <img src="images/langusges/html.png" alt="js" />
-                <p>HTML</p>
-                </li>
-                <li class="lang">
-                <img src="images/langusges/css.png" alt="js" />
-                <p>CSS</p>
-                </li>
-                <li class="frameworks">
-                <p class="langusge">Frameworks</p>
-                <p class="sign">></p>
-                </li>
-                
-                <li class="frameworks">
-                <p class="langusge">Skills</p>
-                <p class="sign">></p>
-                </li>
+                    <li class="lang-title">
+                        <p class="langusge">Languages</p>
+                        <img class="allow-down" src="images/Icons/vector.png" alt="allow-down" />
+                    </li>
+                    <li class="lang">
+                        <img src="images/langusges/js.png" alt="js" />
+                        <p>JavaSript</p>
```

Comment:
- [ ] Kindly ensure that the about myself container follows the design guideline for desktop screens by making the `Languages` align with about myself and likewise fixing the `Get resume` button's margin. 😉

![image](https://user-images.githubusercontent.com/41163571/230603822-2f56a9cb-ff04-4d9f-be32-2072c01e40e1.png)